stdlib/Profile/src/Allocs.jl

```diff
@@ -0,0 +1,180 @@
+module Allocs
+
+using Base.StackTraces: StackTrace, StackFrame, lookup
+using Base: InterpreterIP
+
+# raw results
+
+# matches RawBacktrace on the C side
+struct RawBacktrace
+    data::Ptr{Csize_t} # in C: *jl_bt_element_t
```

Comment:
```suggestion
    data::Ptr{Cvoid} # in C: *jl_bt_element_t
```
I think using `void*` for opaque types is more idiomatic for a C interface.

Reply:
Ah, but i see down below you _aren't_ using this as an opaque pointer, you're actually loading from it directly. It's only working because the `jl_bt_element_t` struct *happens* to be exactly the same amount of bits as `Csize_t`:
https://github.com/JuliaLang/julia/blob/a67bb2bd37f23f2b4ace373c8ea58326034e9bab/src/julia_internal.h#L904-L909

This makes me a bit nervous. Is there some way that we can either use a different type here that's kept in sync with `_jl_bt_element_t`, or make this more explicit?

Reply:
Resolved in https://github.com/vilterp/julia/pull/10